.travis.yml

```diff
@@ -1,22 +1,43 @@
-sudo: false
-language: scala
-addons:
-  hostname: localhost
-jdk:
-  - oraclejdk8
-scala:
-  - 2.11.12
-  - 2.12.8
-before_install:
-  - docker pull daunnc/openjdk-gdal:2.4.0
+# Required to clean up sbt caches written by container image as root
+sudo: required
+
+# https://github.com/travis-ci/travis-ci/issues/2856
+language: bash
+
 services:
   - docker
+
+before_install:
+  - docker pull quay.io/azavea/openjdk-gdal:2.3.2-jdk8-slim 
```

Comment:
Why the explicit pull here vs. letting Docker Compose handle it?

Reply:
I was blindly following some examples from the Travis docs and had assumed that there was some caching benefit.